README.md

```diff
@@ -92,6 +120,22 @@ A minimal example of using an existing configuration.nix:
   };
 }
 ```
+</details>
+
+Instead of using `darwin-installer`, you can just run `darwin-rebuild switch` to install nix-darwin. As `darwin-rebuild` won't be installed in your `PATH` yet, you can use the following command:
+
+```bash
+nix run nix-darwin#darwin-rebuild -- switch --flake ~/.config/nix-darwin#Johns-MacBook
+```
+
+After installing, you can run `darwin-rebuild` directly:
+
+```bash
+darwin-rebuild switch --flake ~/.config/nix-darwin#Johns-MacBook
+```
```

Comment:
These make sense if the reader expanded the migration section, but not for the from-scratch one. I suggest adding wording to the from-scratch section telling the reader to replace `simple` with their system's hostname, as is best practice, and omitting the `#...` here.